src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -851,47 +851,6 @@ class StubGenerator: public StubCodeGenerator {
 
   Label copy_f, copy_b;
 
-  // All-singing all-dancing memory copy.
-  //
-  // Copy count units of memory from s to d.  The size of a unit is
-  // step, which can be positive or negative depending on the direction
-  // of copy.  If is_aligned is false, we align the source address.
-  //
-  /*
```

Comment:
Why the removal of that comment?

Reply:
> Why the removal of that comment?

Thanks for the review, this comment does not match the actual assembly logic, where the align logic of source and dst address  is judged first, not only align the source address. 
https://github.com/openjdk/jdk/blob/b412fc79c3c2548df10918090beedaf6b2d08d96/src/hotspot/cpu/riscv/stubGenerator_riscv.cpp#L987-L993